rules/integrations/cloud_defend/execution_file_made_executable_via_chmod_inside_a_container.toml

```diff
@@ -0,0 +1,54 @@
+[metadata]
+creation_date = "2023/04/26"
+integration = ["cloud_defend"]
+maturity = "production"
+min_stack_comments = "New Integration: Cloud Defend"
+min_stack_version = "8.8.0"
+updated_date = "2023/04/26"
+
+[rule]
+author = ["Elastic"]
+description = "This rule detects when chmod is used to add the execute permission to a file inside a container. Modifying file permissions to make a file executable could indicate malicious activity, as an attacker may attempt to run unauthorized or malicious code inside the container."
+from = "now-6m"
+index = [ "logs-cloud_defend*" ]
+interval = "5m"
+language = "eql"
+license = "Elastic License v2"
+name = "File Made Executable via Chmod Inside A Container"
+risk_score = 47
+rule_id = "ec604672-bed9-43e1-8871-cf591c052550"
+severity = "medium"
+tags = [
+  "Elastic",
+  "Host",
+  "Linux",
+  "Threat Detection",
+  "Execution",
+  "Container"
+]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = """
+process where process.entry_leader.entry_meta.type== "container" and event.type== "start" and 
+(
+(process.name : "chmod" and process.args : "*x*") or 
+/*account for tools like busybox, coreutils that package various utilities into a single executable*/
+(process.args: "chmod" and process.args: "*x*")
+)
+/*not a newly initiated container*/
+and not (process.previous.args: "runc" and process.previous.args: "init")
```

Comment:
Why this condition?

Reply:
Trying to avoid capturing instances where chmod +x is part of the initial container creation process but after getting clarity from DFC team this may not be a very reliable means of capturing a newly created container. I will remove this exception from the rule for now.